src/main/target/SPEEDYBEEF7V3/target.h

```diff
@@ -149,6 +149,7 @@
 // ********** Optiical Flow adn Lidar **************
 #define USE_RANGEFINDER
 #define USE_RANGEFINDER_MSP
+#define RANGEFINDER_I2C_BUS         BUS_I2C2
```

Comment:
This should probably be I2C1, same as MAG.

But the original submitter has updated his pull request.